src/remote/activitypub/kernel/create/index.ts

```diff
@@ -11,6 +12,22 @@ export default async (actor: IRemoteUser, activity: ICreate): Promise<void> => {
 
 	logger.info(`Create: ${uri}`);
 
+	// copy audiences between activity <=> object.
+	if (typeof activity.object === 'object') {
+		const to = unique(concat([toArray(activity.to), toArray(activity.object.to)]));
```

Comment:
prelude の `concat` 廃止で `Array.prototype.flat` でいい気もする。（その方が個人的には読みやすい）

Example
--

```suggestion
		const to = unique([toArray(activity.to), toArray(activity.object.to)].flat());
```

Reply:
たぶん今の設定だと？`.flat()`ないって怒られちゃいます
![image](https://user-images.githubusercontent.com/30769358/74083803-5be7c380-4aab-11ea-9426-881889897277.png)

Reply:
Node.js v11 以降は実装されていたはずなので、設定変えてもいいかもしれないですね。

Reply:
まあ別でやりましょう。